src/test/routes/sign-in.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { app } from '../../main/app';
+
+const options = ['hmcts', 'common', 'pi'];
+const externalUrls = ['https://www.google.com','https://www.google.com','https://www.google.com'];
+
+describe('Sign In option', () => {
+  describe('on GET', () => {
+    test('should return sign-in routing page', async () => {
+      await request(app)
+        .get('/sign-in')
+        .expect((res) => expect(res.status).to.equal(200));
+    });
+  });
+
+  for (let i = 0; i < options.length; i += 1) {
```

Comment:
👍

Reply:
I think this can be `i++` but is really minor, won't block you for this

Reply:
changed to i++